core/arch/arm/plat-ti/main.c

```diff
@@ -175,3 +176,24 @@ void console_init(void)
 			     CONSOLE_UART_CLK_IN_HZ, CONSOLE_BAUDRATE);
 	register_serial_console(&console_data.chip);
 }
+
+#if defined(CFG_OTP_SUPPORT)
+
+void tee_otp_get_hw_unique_key(struct tee_hw_unique_key *hwkey)
+{
+	memcpy(&hwkey->data[0], &plat_huk[0], sizeof(hwkey->data));
+}
+
+/* TODO */
```

Comment:
A patch such as this one [0] would remove the need for this temporary tee_otp_get_die_id clone.

[0] https://github.com/OP-TEE/optee_os/pull/1318/commits/32dce12ce6580ebb332f8a73efe2f80d48d81d39